src/NServiceBus.Unity/UnityObjectBuilder.cs

```diff
@@ -140,8 +142,6 @@ public bool HasComponent(Type componentType)
 
         public void Release(object instance)
         {
-            //Not sure if I need to call this or not!
-            container.Teardown(instance);
```

Comment:
Something changed i v5 to make this not needed?

Reply:
Teardown has been removed and apparently did nothing with the built in strategies anyway. https://github.com/unitycontainer/container/issues/32